### PR TITLE
Merge to main for build17 release (registry v1.7.0)

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary

We want to get ready for the deployment in Test from the main branch, 
then in prod, from a tag.

### 🤖 AI Assistance Disclosure
- [ ] No AI assistance used
- [ ] AI used for light assistance (e.g., suggestions, refactoring, documentation help, minor edits)
- [ ] AI used for moderate content generation (AI generated some code or logic, but the developer authored or heavily revised the majority)
- [x] AI generated substantial portions of this code

Estimated % of code influenced by AI: _80_ %

## ⚙️ Test Data and/or Report

Deployment validated in dev venue

## 🤓 Reviewer Checklist

*Reviewers: Please verify the following before approving this pull request.*

### Documentation and PR Content
- [ ] **Documentation:** README, Wiki, or inline documentation (Sphinx, Javadoc, Docstrings) have been updated to reflect these changes.
- [ ] **Issue Traceability:** The PR is linked to a valid GitHub Issue
- [ ] **PR Title:** The PR title is "user-friendly" clearly identifying what is being fixed or the new feature being added, that if you saw it in the Release Notes for a tool, you would be able to get the gist of what was done.

### Security & Quality
- [ ] **SonarCloud:** Confirmed no new High or Critical security findings.
- [ ] **Secrets Detection:** Verified that the Secrets Detection scan passed and no sensitive information (keys, tokens, PII) is exposed.
- [ ] **Code Quality:** Code follows organization style guidelines and best practices for the specific language (e.g., PEP 8, Google Java Style).

### Testing & Validation
- [ ] **Test Accuracy:** Verified that test data is accurate, representative of real-world PDS4 scenarios, and sufficient for the logic being tested.
- [ ] **Coverage:** Automated tests cover new logic and edge cases.
- [ ] **Local Verification:** (If applicable) Successfully built and ran the changes in a local or staging environment.

### Maintenance
- [ ] **Backward Compatibility:** Confirmed that these changes do not break existing downstream dependencies or API contracts (or that breaking changes are clearly documented).

### CODE REVIEW

File: docker/.env [hunk 1]
```diff
@@ -32,12 +32,12 @@ REG_API_APP_PROPERTIES_FILE=./default-config/application.properties
...
 
 # Docker image of the Registry Loader
-REG_LOADER_IMAGE=nasapds/registry-loader:1.2.4
+REG_LOADER_IMAGE=nasapds/registry-loader:latest
 
 # --------------------------------------------------------------------
 # Registry Sweepers
 # --------------------------------------------------------------------
-REG_SWEEPERS_IMAGE=nasapds/registry-sweepers:1.5.0
+REG_SWEEPERS_IMAGE=nasapds/registry-sweepers:latest
```
Comment (jordanpadams) [inline docker/.env:40]: @tloubrieu-jpl based upon the issue we just encountered with sweepers, we should probably pin this to a release on main.

**Review by jordanpadams — APPROVED** [review]
